src/query/service/src/pipelines/executor/executor_graph.rs

```diff
@@ -241,7 +241,16 @@ impl ExecutingGraph {
                 if state_guard_cache.is_none() {
                     state_guard_cache = Some(node.state.lock().unwrap());
                 }
-                let processor_state = match node.processor.event()? {
+                let event = node.processor.event()?;
+                if tracing::enabled!(tracing::Level::TRACE) {
+                    tracing::trace!(
+                        "node id:{:?}, name:{:?}, event: {:?}",
```

Comment:
```suggestion
                        "node id: {:?}, name: {:?}, event: {:?}",
```

Simple change, but more beautiful.